src/gui/qgsmapcanvas.cpp

```diff
@@ -1823,7 +1823,14 @@ void QgsMapCanvas::setSelectionColor( const QColor &color )
     return;
 
   mSettings.setSelectionColor( color );
-
+  QgsDebugMsg( "XXXXXXXXXXXXXXXXX" );
```

Comment:
?

Reply:
Removed. It's my old fashioned / messy debugging style :)